scripts/gha/build_testapps.py

```diff
@@ -558,24 +571,80 @@ def _build_ios(
     _run(
         xcodebuild.get_args_for_build(
             path=xcode_path,
-            scheme=api_config.scheme,
+            scheme=api_config.ios_scheme,
             output_dir=build_dir,
-            ios_sdk=_IOS_SDK_SIMULATOR,
+            apple_platfrom=_IOS,
+            apple_sdk=_IOS_SDK_SIMULATOR,
             configuration="Debug"))
 
   if _IOS_SDK_DEVICE in ios_sdk:
     _run(
         xcodebuild.get_args_for_build(
             path=xcode_path,
-            scheme=api_config.scheme,
+            scheme=api_config.ios_scheme,
             output_dir=build_dir,
-            ios_sdk=_IOS_SDK_DEVICE,
+            apple_platfrom=_IOS,
+            apple_sdk=_IOS_SDK_DEVICE,
             configuration="Debug"))
 
     xcodebuild.generate_unsigned_ipa(
         output_dir=build_dir, configuration="Debug")
 
 
+def _build_tvos(
```

Comment:
Is this different enough from iOS to require a separate function? Or could it just be a parameter to the ios one?

Reply:
The main differences are: 1) Currently, tvOS won't build .ipa for real device. 2) iOS entitlements introduce errors in tvOS apps.
But, yeah, we could add a few more parameters to reuse the iOS one.

Reply:
Interesting. Do we know why the entitlement file is not working on tvos? auth, messaging and dynamic links are using entitlement files on iOS. Am surprised auth is working without it on tvOS. We do not care about dynamic links but messaging might need it once we have it up and running on tvos.

Reply:
Reused the same function with more parameters.
It seems entitlements do not affect current tests in our workflow.
But, we still need to handle this entitlements stuff later. (Probably in other PR.)

Reply:
Sounds good. Maybe we can leave a TODO in code and take out the commented out code for entitlements for tvos?